diff --git a/node_modules/@types/express-session/index.d.ts b/node_modules/@types/express-session/index.d.ts
index 0d302e3..4877225 100644
--- a/node_modules/@types/express-session/index.d.ts
+++ b/node_modules/@types/express-session/index.d.ts
@@ -14,7 +14,7 @@ declare global {
              *
              * @see SessionData
              */
-            session: session.Session & Partial<session.SessionData>;
+            session: any;
 
             /**
              * This request's session ID.
